prime-router/src/test/resources/fhirengine/translation/FHIR_to_HL7/input.fhir

```diff
@@ -267,6 +267,17 @@
             }
           ]
         },
+        "extension" : [
```

Comment:
This caused a couple of [TranslationSchemaManager unit tests](https://github.com/CDCgov/prime-reportstream/blob/6a92b973bad95403ff08e4eb3646e19c77603d64/prime-router/src/test/kotlin/fhirengine/translation/TranslationSchemaManagerTests.kt#L835) to fail because they are not using the simple report sender schema.